includes/class-wsuwp-alumni-awards.php

```diff
@@ -271,4 +272,142 @@ public function admin_enqueue_scripts( $hook_suffix ) {
 			wp_enqueue_style( 'alumni-awards-admin', plugins_url( 'css/admin.css', dirname( __FILE__ ) ) );
 		}
 	}
+
+	/**
+	 * Display alumni awardees.
+	 *
+	 * @since 0.0.1
+	 *
+	 * @param array $atts List of attributes passed to the shortcode.
+	 *
+	 * @return string Content to display for the shortcode.
+	 */
+	public function display_alumni_awards( $atts ) {
+		$defaults = array(
+			'award_slug' => '',
+			'awarded' => '',
+			'class' => '',
+			'filters' => '',
+			'header' => '',
+			'sport' => 'hide',
+			'type' => 'Recipient',
+			'inscription' => '',
+		);
+
+		$atts = shortcode_atts( $defaults, $atts );
+
+		// The `award_slug` attribute must be set.
+		if ( '' === $atts['award_slug'] ) {
+			return '<!-- No award_slug attribute set -->';
+		}
+
+		wp_enqueue_style( 'alumni-awards', plugins_url( 'css/shortcode.css', dirname( __FILE__ ) ) );
```

Comment:
I'd add a version here to make it easier to break cache in the future if necessary.